tests/mpi/hp_step-40.with_petsc=true.mpirun=10.output.2

```diff
@@ -0,0 +1,17 @@
+
+Cycle 0:
+   Number of active cells:       1024
+      104+100+104+100+104+104+100+104+100+104+
+   Number of degrees of freedom: 4225
+      465+416+432+416+416+432+416+416+400+416+
+DEAL:mpi::Solver stopped within 120 - 260 iterations
```

Comment:
the iterations numbers are completely different here in comparison with the alternative output, why?

Reply:
I have no idea -- that's what `mpi/step-40` already does. Do you want me to modify that test as well?